float64 lat
float64 lon
float64 alt
float64 break_alt
float64 land_dir
uint8 flags
